src/utils.ts

```diff
@@ -12,6 +12,14 @@ export type Decimalish = string | number | Decimal;
 export const AddressZero = ethers.constants.AddressZero;
 
 const { ConvertDecimals } = uma.utils;
+const BlockSpeedSample: { [chainId: number]: number } = {
+  1: 10000,
```

Comment:
How did you pick these samples by the way?

Reply:
These don't have to exact and are meant to be a starting point. The way the algo works here is it fetches the first block exactly 1 distance away from the latest block and uses the timestamp difference to calculate the initial block speed. From then on, it'd add a multiplier to try to get as close to the desired timestamp as possible. The heuristic here is to have these be relatively small but not too small as that'd lead to the first step getting a volatile speed and just make the search less efficient. On the other hand, if these values are too large and the desired lookback timestamp is small, we might end up going back too far.